<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialConfigurationView</class>
 <widget class="QWidget" name="SerialConfigurationView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>cmd:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="message"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="portHorizontalLayout">
       <item>
        <widget class="QLabel" name="portLabel">
         <property name="text">
          <string>Port:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="portBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="dataBitsHorizontalLayout">
       <item>
        <widget class="QLabel" name="dataBitsLabel">
         <property name="text">
          <string>DataBits:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="dataBitsBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="parityHorizontalLayout">
       <item>
        <widget class="QLabel" name="parityLabel">
         <property name="text">
          <string>Parity:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="parityBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="stopBitsHorizontalLayout">
       <item>
        <widget class="QLabel" name="stopBitsLabel">
         <property name="text">
          <string>StopBits:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="stopBitsBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="baudRateHorizontalLayout">
       <item>
        <widget class="QLabel" name="baudRateLabel">
         <property name="text">
          <string>BaudRate:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="baudRateBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="queryModeHorizontalLayout">
       <item>
        <widget class="QLabel" name="queryModeLabel">
         <property name="text">
          <string>QueryMode:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="queryModeBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="timeOutHorizontalLayout">
       <item>
        <widget class="QLabel" name="timeOutLabel">
         <property name="text">
          <string>Timeout:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="timeOutSpinBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="HLed" name="led" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="openCloseButton">
         <property name="text">
          <string>Open/Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Response:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="received_msg"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HLed</class>
   <extends>QWidget</extends>
   <header>hled.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
